PrototopeJSBridge/BehaviorBridge.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  BehaviorBridge.swift
+//  Prototope
+//
+//  Created by Saniul Ahmed on 11/02/2015.
+//  Copyright (c) 2015 Khan Academy. All rights reserved.
+//
+
+import Foundation
+import JavaScriptCore
+import Prototope
+
+func behaviorBridgeForBehavior(behavior: BehaviorType) -> BehaviorBridgeType {
+    switch behavior {
+    case is CollisionBehavior: return CollisionBehaviorBridge(behavior as! CollisionBehavior)
+    case is BlockBehavior: return BlockBehaviorBridge(behavior as! BlockBehavior)
+        
+    default: abort()
+    }
+}
+
+func behaviorForBehaviorBridge(behaviorBridge: BehaviorBridgeType) -> BehaviorType {
+    switch behaviorBridge {
+    case is CollisionBehaviorBridge: return (behaviorBridge as! CollisionBehaviorBridge).collisionBehavior
+    case is BlockBehaviorBridge: return (behaviorBridge as! BlockBehaviorBridge).blockBehavior
+    default: abort()
+    }
+}
+
+@objc public protocol BehaviorBridgeType {
+}
+
+@objc public protocol CollisionBehaviorJSExport: JSExport {
+    init?(args: JSValue)
+}
+
+//MARK: CollisionBehavior
+
+@objc public class CollisionBehaviorBridge: NSObject, BridgeType, BehaviorBridgeType, CollisionBehaviorJSExport {
+    let collisionBehavior: Prototope.CollisionBehavior!
+    
+    public class func addToContext(context: JSContext) {
+        context.setObject(self, forKeyedSubscript: "CollisionBehavior")
+    }
+    
+    public init(_ behavior: CollisionBehavior) {
+        self.collisionBehavior = behavior
+        super.init()
+    }
+    
+    required public init?(args: JSValue) {
+        let kindValue = args.valueForProperty("kind")
+        let otherLayerValue = args.valueForProperty("otherLayer")
+        let handler = args.objectForKeyedSubscript("handler")
```

Comment:
This actually needs to be `args.valueForProperty("handler")` because you need to leave it as a `JSValue` to `callWithArguments` on line 61.